tests/functions.py

```diff
@@ -0,0 +1,76 @@
+from os import listdir, walk
+from os.path import isdir, exists, join, abspath
+from random import random
+import datalad.api as api
+import subprocess
+
+
+def recurse(directory, odds):
+
+    # Get all file names in directory
+    files = listdir(directory)
+
+    # Loop throw every file
+    for file in files:
+
+        # Execute any config.sh scripts
+        if file == "config.sh":
+            with open(file, "r") as f:
+                line = f.readline()
+                while line:
+                    subprocess.call(line)
+                    line = f.readline()
+            continue
```

Comment:
Replace by a line of bash in `.travis.yml`, executed before the tests are run: something like `find . -name config.sh | xargs bash`.